src/snowflake/connector/connection.py

```diff
@@ -186,6 +187,10 @@ def DefaultConverterClass():
     ),  # only use openssl instead of python only crypto modules
     # whether to convert Arrow number values to decimal instead of doubles
     "arrow_number_to_decimal": (False, bool),
+    "enable_stage_s3_privatelink_for_us_east_1": (
```

Comment:
Add getter and maybe a setter for `enable_stage_s3_privatelink_for_us_east_1` to return `_enable_stage_s3_privatelink_for_us_east_1`